packages/orchestrator/main.go

```diff
@@ -149,6 +155,31 @@ func main() {
 		}
 	}()
 
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+
+		errChan := make(chan error, 1)
+		go func() {
+			errChan <- sessionProxy.Start()
+		}()
+
+		select {
+		case <-ctx.Done():
+		case err = <-errChan:
+			if err != nil {
+				zap.L().Error("session proxy failed", zap.Error(err))
+				exitCode.Add(1)
+				cancel()
+			}
+		}
+
+		// close session proxy, wait 5 seconds until all connections are closed
+		shutdownCtx, shutdownCtxCancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
this can be up to 24 hours (sandbox max lifetime)

Reply:
in client-proxy there is for server shutdown function used `context.Background()` should we use that? Like it maybe more than just explicitly define 24 hours timeout.

Will also add log telling server shutdown started to make it more clear what is happening during shutdown.